packages/flutter_test/lib/src/controller.dart

```diff
@@ -2222,11 +2222,14 @@ abstract class WidgetController {
     Duration duration = const Duration(milliseconds: 50),
   }) {
     return TestAsyncUtils.guard<void>(() async {
+      TestGesture? gesture;
       while (maxIteration > 0 && finder.evaluate().isEmpty) {
-        await drag(view, moveStep);
+        gesture ??= await startGesture(getCenter(view, warnIfMissed: true));
+        await gesture.moveBy(moveStep);
```

Comment:
There are quite a lot of google testing failures. I wonder if there are things in the `drag` method that this refactor doesn't replace?

Reply:
I'm guessing the tests this affects in Google were written to expect the behavior written in the bug. This might be difficult to change since it breaks existing test logic.